Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -2194,23 +2196,24 @@
         <AbilArray Link="AP_GhostSCBWLockdown"/>
         <AbilArray Link="AP_GhostResourceEfficiencyMorph"/>
         <BehaviorArray Link="AP_OcularImplantsIndicator"/>
-        <BehaviorArray Link="AP_CriusSuitIndicator"/>
         <BehaviorArray Link="AP_GhostResourceEfficiencyMorph"/>
+        <BehaviorArray Link="AP_GhostPermanentCloak"/>

```

Comment:
Why not to just replace the row with the other stuff. Then you don't have to touch the Resource Efficiency variant